vercel/examples
# Add an example to lazy load ads scripts and combine a skeletton to avoid layout shift

### Description



Shows how to lazy load ads with a skeleton and delay loading ad scripts to get pertfect lighthouse score.

[readme](https://github.com/vercel/examples/tree/feat/script-ad/solutions/script-component-ad)

### Best Way to Test

1. turn off ad blockers or this won't work
2. navigate to the page. Pay attention to ad getting loaded and skeletton.
3. run lighthouse to ensure metrics are all green


### Type of Change


### New Example Checklist
